packages/application-shell/src/components/quick-access/quick-access.spec.js

```diff
@@ -409,7 +408,7 @@ describe('QuickAccess', () => {
     );
 
     // should close quick access
-    expect(container).toBeEmpty();
+    expect(queryByTestId('quick-access')).not.toBeInTheDocument();
   });
 
   it('should cycle through the history', async () => {
```

Comment:
@dferber90 this test apparently is currently failing on `master` (at least locally for me).

<img width="686" alt="image" src="https://user-images.githubusercontent.com/1110551/49275333-4c1e2980-f47b-11e8-906c-a9298dfacee1.png">

Can you check if that works for you? In travis it also seem to pass...

Reply:
Weird that it would fail locally but work on CI. I believe we are no longer able to use `toBeEmpty` since we always add the modal container now.

Reply:
Is it failing locally for you as well?

> I believe we are no longer able to use toBeEmpty since we always add the modal container now

Correct, but I think it's better to not rely on an empty container.